### PR TITLE
Add message to play audio note

### PR DESCRIPTION
**Pull request by jdorich14**

### CODE REVIEW

File: msg/AudioNote.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Play a note (frequency) for a specific duration
```
Comment (alsora) [inline msg/AudioNote.msg:1]: I think we should also add a `std_msgs/Header header` field.
Although we don't need it now, it may be useful in the future

Comment (jdorich14) [reply msg/AudioNote.msg:1]: added in 7e3571b77922d018e722baf1b793eb1ca8bb362b

**Review by alsora — APPROVED** [review]

**Comment by justinIRBT** [comment]
Did we want an AudioNoteSequence message as well that is a vector of AudioNote?  Assuming we go that route, should the append flag be in the AudioNoteSequence instead of AudioNote

**Review by alsora — APPROVED** [review]

File: msg/AudioNote.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+uint16_t frequency    # Frequency of note in Hz.
```
Comment (justinIRBT) [inline msg/AudioNote.msg:2]: There is no _t in the ros2 message datatype specification, it should just be `uint16`
https://docs.ros.org/en/foxy/Concepts/About-ROS-Interfaces.html#field-types

**Review by justinIRBT — CHANGES_REQUESTED** [review]

File: msg/AudioNote.msg (hunk 2 above)
Comment (justinIRBT) [reply msg/AudioNote.msg:2]: You also need to add your messages to the CMakeLists.txt in the `set(msg_files`

**Review by justinIRBT — APPROVED** [review]

**Pull request merged**